components/settings/FontSizeCounterInput.tsx

```diff
@@ -1,11 +1,11 @@
 import { useEffect, useState } from "react";
-import CounterInput from "@/components/CounterInput";
+import CounterInput from "../CounterInput";
 
 export default function FontSizeCounterInput() {
 	const [fontSize, setFontSize] = useState(16);
 
 	useEffect(() => {
-		setFontSize(parseInt(localStorage.getItem("fontSize")) || 16);
+		setFontSize(parseInt(JSON.parse(localStorage.getItem("fontSize"))) || 16);
```

Comment:
why parse it as json? I can't see a reason...

Reply:
noted